vnext/Microsoft.ReactNative/IReactDispatcher.idl

```diff
@@ -53,5 +53,23 @@ namespace Microsoft.ReactNative
       "Generally you can use @IReactContext.JSDispatcher to get the value of this property for "
       "a specific React instance.")
     static IReactPropertyName JSDispatcherProperty { get; };
+
+    DOC_STRING(
```

Comment:
>DOC_STRING [](http://example.com/codeflow?start=4&length=10)

We need to change the new comments saying that these are notification names to be used with IReactNotificationService.

Reply:
I have changed the comments, please let me know your thoughts.

Reply:
The first line of each comment still says that it is a property for IReactPropertyBag.

Reply:
I changed the comments in the latest commit. Please review.